### ISSUE TITLE
Scaffold of barebone tests when scaffolding new components to encourage TDD

### ISSUE THREAD

**Issue by 8ball030** [valory-xyz/open-aea#5]
When scaffolding new components, could we scaffold very simple tests for the components.

**Describe the solution you'd like**
i.e. 
scaffold skill new_skill -> skill is scaffolded as is -> simple tests are generated and saved along side the component.

**Comment by DavidMinarsch** [comment]
Yes that's a great one! I'd like to first improve the test tools a bit and then do this as step two. But definitely needed and of high value!

**Comment by DavidMinarsch** [comment]
Closing as obsolete.

The gap is real — scaffold templates under `aea/skills/scaffold/`, `aea/connections/scaffold/`, `aea/contracts/scaffold/`, and `aea/protocols/scaffold/` still ship without test stubs. But the premise for baking test scaffolding into the CLI has aged out:

- Backfilling barebones tests for a freshly scaffolded component is now a trivial LLM prompt. That path is faster, produces tests tailored to the actual component the user is writing, and doesn't rely on template stubs drifting against the current `aea.test_tools` APIs.
- Bundled test templates would need ongoing maintenance every time skill/connection/contract base classes evolve — a recurring cost for a niche entry point (raw `aea scaffold ...` sees little use compared to higher-level tooling in the Olas stack).

Net: the effort is not justified for the audience that still uses `aea scaffold` directly, and the "encourage TDD" goal is better served by the user pointing an LLM at the scaffolded files after creation.
